packages/api/internal/orchestrator/cache.go

```diff
@@ -179,7 +181,9 @@ func (o *Orchestrator) getInsertInstanceFunction(ctx context.Context, logger *za
 			node.CPUUsage.Add(info.VCpu)
 			node.RamUsage.Add(info.RamMB)
 
-			o.dns.Add(info.Instance.SandboxID, node.Info.IPAddress)
+			if err := o.dns.Add(info.Instance.SandboxID, node.Info.IPAddress); err != nil {
```

Comment:
We shouldn't return early here, we should collect errors and then join them

Reply:
The function shouldn't even return error probably, we should just log it here. It's confusing

Reply:
That's an interesting idea, but would the error be logged by the caller of `getInsertInstanceFunction`?

Reply:
Done as requested.